Cargo.toml

```diff
@@ -15,7 +15,7 @@ travis-ci = { repository = "pingcap/rust-prometheus" }
 
 [features]
 default = []
-dev = ["clippy"]
+dev = []
```

Comment:
You can remove this feature now. :)

Reply:
Ok.  is it safe to also remove these two lines on the Makefile? I had no idea since I had not use any Makefile yet😂
https://github.com/pingcap/rust-prometheus/blob/master/Makefile#L11

Reply:
You can remove the dev feature and make `make dev` to be `format + test`.

Reply:
done